src/main/java/com/ably/kafka/connect/DefaultChannelSinkMapping.java

```diff
@@ -0,0 +1,76 @@
+package com.ably.kafka.connect;
+
+import io.ably.lib.realtime.AblyRealtime;
+import io.ably.lib.realtime.Channel;
+import io.ably.lib.types.AblyException;
+import io.ably.lib.types.ChannelMode;
+import io.ably.lib.types.ChannelOptions;
+import org.apache.kafka.connect.sink.SinkRecord;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import javax.annotation.Nonnull;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import static com.ably.kafka.connect.ChannelSinkConnectorConfig.*;
+
+public class DefaultChannelSinkMapping implements ChannelSinkMapping {
```

Comment:
There's some inconsistency in method names in this class. While I recognise some are `private`, I still can't understand why `getAblyChannelName()` and `getAblyChannelOptions()` both have `Ably` in their name, where `getChannel(SinkRecord, AblyRealtime)` doesn't. Plus, of course, `convertChannelParams(List<String>)` doesn't have `Ably` in its name.

I generally try to go for "less is more" with naming, taking into account context... i.e.:
- we already know this is `Ably` because package name is `com.ably.kafka.connect`, so safe to drop our name from names if possible
- we already know this relates to channels because the class has `Channel` in its name, as well as the name of the interface it implements

Reply:
These methods were carried over from Connector class in order to produce configurations for individual channels without changing their names, etc. I was actually planning to move these into another converter class in next set of refactorings. I will create an issue from your comment to remind myself about change.